lib/elixir/test/elixir/exception_test.exs

```diff
@@ -50,6 +50,19 @@ defmodule ExceptionTest do
                {:io, :put_chars, [self(), <<222>>],
                 [error_info: %{module: __MODULE__, function: :dummy_error_extras}]}
              ])
+
+    assert %ErlangError{
+             original: {:failed_load_cacerts, :enoent},
+             reason: ": Failed to load cacerts: operating system CA bundle could not be located"
+           } =
+             Exception.normalize(:error, {:failed_load_cacerts, :enoent}, [
+               {:pubkey_os_cacerts, :get, 0,
+                [
+                  file: ~c"pubkey_os_cacerts.erl",
+                  line: 53,
+                  error_info: %{cause: :enoent, module: :pubkey_os_cacerts}
+                ]}
+             ])
```

Comment:
Given this will depend on the Erlang/OTP version, I wonder if we should write our own Erlang error that reproduces the failure?

Reply:
Woops right, there was an example just above reused the same approach: [4c61768](https://github.com/elixir-lang/elixir/pull/14329/commits/4c617683d05c37c28e2555bf5dbb4d6a0876fc2d)